wolfSSL-FIPS-Ready/src/section01.md

```diff
@@ -0,0 +1,60 @@
+# wolfSSL FIPS Ready
+
+Do you have a project you'll need for a FIPS approved cryptographic library at 
+a future date and want to be ready for it now? wolfSSL FIPS Ready is exactly 
+what you need. It is the wolfSSL Inc FIPS enabled cryptography layer code 
+included in the wolfSSL source tree that one can enable and build. You do not 
+get a FIPS certificate, you are not FIPS approved, but you will be Ready to go 
+through the FIPS process when the time comes. FIPS Ready means that you have 
+included the FIPS code into your build and that you are operating according to 
+the FIPS enforced best practices of default entry point, and conditional 
+algorithm self-tests (CAST) that came with FIPS 140-3. When the time comes, 
+you can get your operating environment tested and added to the wolfSSL Inc 
+FIPS 140-3 certificate for wolfCrypt or submit for a brand new FIPS 140-3 cert 
+from scratch and all the coding work will have been done.
```

Comment:
Need a little more run-on sentence cleanup, something like: "When the time comes, we will shepherd your operating environment through testing, and get it added to the existing wolfCrypt FIPS 140-3 certificate.  In some situations, a new FIPS 140-3 certificate will be required, and we will work with you through the testing and certification process.  In both scenarios, FIPS Ready assures that your application will be fully functional with wolfCrypt as certified."